src/plugins/index.ts

```diff
@@ -149,7 +160,7 @@ export class TransparentPlugins implements KeyToPluginInstance {
   get textMode () { return this.ep["text-mode"]; }
   get performanceInfo () { return this.ep["performance-info"]; }
   get metadata () { return this.ep["manage-metadata"]; }
-  get intellisense () { return this.ep["intellisense"]; }
+  get multiline () { return this.ep["multiline"]; }
```

Comment:
Intellisense shouldn't be removed here in the diff.

Reply:
Fixed!